json/targeting.json

```diff
@@ -4,6 +4,11 @@
   "title": "flagd Targeting",
   "description": "Defines targeting logic for flagd; a extension of JSONLogic, including purpose-built feature-flagging operations.",
   "type": "object",
+  "properties": {
+    "targeting": {
+      "$ref": "#/definitions/targeting"
+    }
+  },
```

Comment:
```suggestion
 "anyOf": [
   {
     "$ref": "#/definitions/targeting"
   }
 ],
```

Reply:
Oh you're totally right. I've corrected this and updated all the tests!